File: def_fucntion/Fibonacci.py
def fibonacci(n): # write Fibonacci series up to n
    """Print a Fibonacci series up to n """
    a, b = 1, 2
    while a <= n:
        print(a, end= ' ')
        a, b = b, a + b
    print()

fibonacci(2000)
print(fibonacci(1000))

File: def_fucntion/Fibonacci.java
public class Fibonacci {

    public static void main(String[] args) {
        fibonacci(2000);
    }

    static void fibonacci(int n) {
        if (n <= 0) {
            System.out.println("Illegal paramter: n <=0");
            return;
        }

        int a = 1, b = 2;
        while (a <= n) {
            System.out.print(a + " ");
            int c = a + b;
            a = b;
            b = c;
        }

        System.out.println();
    }
}
